### ISSUE TITLE
Right-click to Copy-Paste anything

### ISSUE THREAD

**Issue by brandontan903** [beaverbuilder/feature-requests#75]
I'm always frustrated when designing pages with full of contents when I need to change the button all over the page, I need to duplicate the button and drag it so far to many place.

Dragging a button when designing page with full of contents is very frustrating. Every time this happen, I pray for Beaver Builder to be better one day.

Currently I'm using 2 page builders. Just in case I need to design quicker, I'm using another page builder instead of Beaver Builder.

**The solution**

Right-click to copy & paste anything is the best solution by far. Elementor, Divi, Brizy all have right-click to copy & paste.

Also important, right-click to paste style.

**Comment by quasel** [comment]
Alternatives: 
Open the Module / Advanced and Copy&Paste sadly this only works on Module level 
![image](https://user-images.githubusercontent.com/9464217/98815281-31a2a800-2427-11eb-9982-6349e4d0f607.png)

Or get https://sitespot.dev/product/beaver-team-pro/ - they ad an "update all" button to apply changes to all Buttons on same site or same category or…

**Comment by jodzeee** [comment]
I love the idea of an "Update All" solution!

**Comment by gbissland** [comment]
A 'quicker' copy & paste would be useful for sure! and some sort of Global styles or updates.

For buttons, I have the styles setup in my starter theme css and use classes for any diff colors.  
Seems to be a faster build and is easy to update.

**Comment by jodzeee** [comment]
Yes, that's how I do it. I still use a button class and rarely use the button module. But it would be nice to have the button module have the ability to be in sync with the classes, if that makes sense.

**Comment by codente** [comment]
FYI: We are looking at improving copy and paste and seeing what our options are currently.  This is being considered for 2.7 and possibly 2.6 if it's ready.

**Comment by codente** [comment]
Improved copy and paste was added in 2.6 :)

**Comment by zackpyle** [comment]
@codente I would love to see an option for Copy/Paste Styles in addition to the Copy/Paste Settings. This is a lot more common for me to need to duplicate the styles from one to another. If I wanted to copy all settings, then I could just duplicate the module.
![Screen Shot 2023-03-30 at 13 49 20](https://user-images.githubusercontent.com/19413506/228937386-ca9e0d30-360f-4d1a-8e5b-7fe6af184fe9.jpg)

**Comment by zackpyle** [comment]
@codente I'm confused. How is this working then??
![Screen Shot 2023-03-30 at 15 27 56](https://user-images.githubusercontent.com/19413506/228956564-0baf3b8c-9c5c-42ed-bffe-6df73770154b.jpg)

**Comment by pixelloop** [comment]
While I'm not an Elementor fan, their right click to copy or copy styles and paste or paste styles is such a rapid way to work. 

Beaver Builder's approach was always pretty good but I would love if they added the ability to copy and paste styles from one module, row or box to another with just with two right clicks and right clicking anywhere on the module.
